src/main/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentController.java

```diff
@@ -145,7 +146,18 @@ public CaseAssignmentResponse assignAccessWithinOrganisation(
     })
     public GetCaseAssignmentsResponse getCaseAssignments(@RequestParam("case_ids")
             @Valid @NotEmpty(message = "case_ids must be a non-empty list of proper case ids.") List<String> caseIds) {
+        validateCaseIds(caseIds);
```

Comment:
this is a defect fix on the fly, as i understand?
Nice if so.

Reply:
yes this was @smathangi fix